sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/NetAppVolumeDataProtection.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Azure.ResourceManager.NetApp.Models;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Azure.ResourceManager.NetApp.Models
+{
+    /// <summary> DataProtection type volumes include an object containing details of the replication. </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public partial class NetAppVolumeDataProtection
```

Comment:
Do we have its generated version previously? Why I did not see it gets renamed to this here?

Reply:
not being renamed
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/netapp/Azure.ResourceManager.NetApp/src/Generated/Models/NetAppVolumeDataProtection.cs

Reply:
I withdraw my previous comment on this class. In the new version is just something new added to this property, we do not need to add anything "internal" back because customers could never use them
Therefore please just remove this file because it does not need any customization.